sdk/search/Azure.Search.Documents/src/Models/SynonymMap.cs

```diff
@@ -7,6 +7,7 @@
 
 namespace Azure.Search.Documents.Models
 {
+    [CodeGenSuppress(nameof(SynonymMap), typeof(string), typeof(string))]
```

Comment:
We don't suppress generated ctors any more if you have a custom one so this have to be done manually.